tests/unit/test_util.py

```diff
@@ -37,32 +37,43 @@ class FakeModel(FakeApiModelExtension):
     name: str
 
 
+class FakeModel2(FakeApiModelExtension):
+    code: int = 0
+    name: str
+    status: str
+
+
 class TestParseInput:
 
     def test_no_data(self, app):
         """
         GIVEN a request with no parameters of any type
         WHEN we call parse_input on that request
-        THEN we get a BaseApiModel returned with a code of 400
+        THEN we get a BaseApiView returned with a code of 400
         AND an error explaining what is wrong
         """
         with app.test_request_context():
             retval = parse_input(app.logger, FakeModel)
-            assert 'Field required' in retval.error
-            assert isinstance(retval, BaseApiModel)
+            assert 'Field required' in retval.errors["parse_input"][0]["msg"]
```

Comment:
I would prefer to have '.get()' used here, for instance `retval.get('parse_input')`. This will return an empty string instead of throwing a keyerror, which I think is easier to debug if the test breaks. However, before changing this part of the tests, we probably need to agree on format of the errors/details key.

Reply:
This makes sense. Thanks for pointing this out!